gregorio-test.sh

```diff
@@ -52,6 +53,12 @@ then
     source $rcfile
 fi
 
+if [[ "$OSTYPE" == "linux-gnu" ]]; then
```

Comment:
The style of this file has been to put "then" on the next line, so it would be nice to keep it that way.